src/index.ts

```diff
@@ -347,7 +348,7 @@ function parseAttributeLine (line: string, lastChange: Changed, errors: Array<Pa
   lastChange.changedAttributes[name] = result;
 }
 
-export function parseStdout (logOutput: string): ParseResult {
+export function parseStdout (logOutput: string, clean: boolean): ParseResult {
```

Comment:
Can you change this to an `options` argument that has a `clean` property? I'm worried that additional options will be added in the future and it will be hard to prevent breaking change if we don't provide this flexibility now.

Reply:
Also, make the second `options` argument optional. For example, something like:

```typescript
function parseStdout (logOutput: string, options?: ParseOptions): ParseResult {
```

Reply:
Sorry for the delay, I was off on a vacation I will get going on making these changes 😄

Reply:
Should all be fixed